frontend/src/scenes/error-tracking/components/ExceptionCard/Stacktrace/StacktraceTextDisplay.tsx

```diff
@@ -1,24 +1,24 @@
 import { LemonSkeleton } from '@posthog/lemon-ui'
-import { useActions, useValues } from 'kea'
+import { useActions, useMountedLogic, useValues } from 'kea'
+import { errorPropertiesLogic } from 'lib/components/Errors/errorPropertiesLogic'
 import { stackFrameLogic } from 'lib/components/Errors/stackFrameLogic'
 import { ExceptionHeaderProps } from 'lib/components/Errors/StackTraces'
 import { ErrorTrackingException, ErrorTrackingStackFrame } from 'lib/components/Errors/types'
-import { hasStacktrace } from 'lib/components/Errors/utils'
 import { cn } from 'lib/utils/css-classes'
 import { useCallback, useEffect } from 'react'
 
+import { exceptionCardLogic } from '../exceptionCardLogic'
 import { StacktraceBaseDisplayProps, StacktraceBaseExceptionHeaderProps } from './StacktraceBase'
 
 export function StacktraceTextDisplay({
     className,
-    attributes,
     renderLoading,
     renderEmpty,
     truncateMessage,
-    loading,
 }: StacktraceBaseDisplayProps): JSX.Element {
-    const { exceptionList } = attributes || {}
-    const isEmpty = !hasStacktrace(exceptionList || [])
+    const logic = useMountedLogic(errorPropertiesLogic)
+    const { exceptionList, hasStacktrace } = useValues(logic)
```

Comment:
If you're using a bound logic higher up you shouldn't need to add the props to pick it up so this will work:
```suggestion
    const { exceptionList, hasStacktrace } = useValues(errorPropertiesLogic)
```

Reply:
Ok good to know